### ISSUE TITLE
loss is negative

### ISSUE THREAD

**Issue by fengjiejiejiejie** [milesial/Pytorch-UNet#84]
Hi, milesial!
the format of my mask is PNG,  yours are GIF, the loss is negative when I was trainning.  
When I changed PNG to GIF, the loss is  positive.
I don't know why,
thank you deeply if you can answer me!

INFO:
    
Epoch 1/5: 100%|██████████| 79/79 [00:12<00:00,  6.33img/s, loss (batch)=-9.27e+5]
Validation round: 100%|██████████| 13/13 [00:01<00:00, 12.87img/s]
INFO: Checkpoint 1 saved !
INFO: Validation Dice Coeff: 1.9721606236237745
Epoch 2/5: 100%|██████████| 79/79 [00:11<00:00,  7.11img/s, loss (batch)=-4.68e+6]
Validation round: 100%|██████████| 13/13 [00:00<00:00, 18.31img/s]
INFO: Validation Dice Coeff: 1.9721606236237745
Epoch 3/5: 100%|██████████| 79/79 [00:11<00:00,  7.12img/s, loss (batch)=-1.14e+7]
Validation round: 100%|██████████| 13/13 [00:00<00:00, 18.49img/s]
INFO: Validation Dice Coeff: 1.9721606236237745
Epoch 4/5: 100%|██████████| 79/79 [00:11<00:00,  7.10img/s, loss (batch)=-2.15e+7]
Validation round: 100%|██████████| 13/13 [00:00<00:00, 18.33img/s]
INFO: Validation Dice Coeff: 1.9721606236237745
Epoch 5/5: 100%|██████████| 79/79 [00:11<00:00,  7.09img/s, loss (batch)=-3.4e+7]
Validation round: 100%|██████████| 13/13 [00:00<00:00, 18.54img/s]
INFO: Validation Dice Coeff: 1.9721606236237745

Process finished with exit code 0

**Comment by milesial** [comment]
Hi, I see also that your Dice coeff is > 1.

It means that when you are loading PNG, you are loading it wrong. If you have 1 class, the masks should be loaded as greyscale (1 channel), with values between 0 and 1 (most often only 0s and 1s)

If there are mask values greater than 1, the Binary Cross Entropy is confused...
Make sure this line https://github.com/milesial/Pytorch-UNet/blob/9fd386f5929945f3df1868930d392507fee16a55/utils/load.py#L19 loads values from 0 to 1.

**Comment by fengjiejiejiejie** [comment]
> Hi, I see also that your Dice coeff is > 1.
> 
> It means that when you are loading PNG, you are loading it wrong. If you have 1 class, the masks should be loaded as greyscale (1 channel), with values between 0 and 1 (most often only 0s and 1s)
> 
> If there are mask values greater than 1, the Binary Cross Entropy is confused...
> Make sure this line
> 
> https://github.com/milesial/Pytorch-UNet/blob/9fd386f5929945f3df1868930d392507fee16a55/utils/load.py#L19
> 
> loads values from 0 to 1.

Thank you so much!
I checked the masks and find the value is 0 or 255, not 1.
